### PR TITLE
added router

### PR DESCRIPTION
**Pull request by keiryan**
Fixed the incorrect routing that would not switch pages.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -10,13 +10,13 @@ import {
...
 import { styled, ThemeProvider, createGlobalStyle } from "styled-components";
-import { Container, MiddleContainer } from "app.styles";
+import { Container } from "app.styles";
 import Home from "pages/Homepage/homepage";
 import Coinpage from "pages/Coinpage/coinpage";
 import Portfolio from "./pages/Portfolio/portfolio";
 import Navbar from "./components/Navbar/navbar";
 import { NavBarNotch } from "components/NavBarNotch/navbarnotch";
-import CoinContainer from "./components/CoinContainer/coincontainer";
+// import CoinContainer from "./components/CoinContainer/coincontainer";
```
Comment (yoloOnTheBattlefield) [inline src/App.js]: no commented out code

File: src/utils/MarketCap/index.js [hunk 2]
```diff
@@ -6,7 +6,6 @@ export function getMarketCap(value){
     //     2: '',
     // }
```
Comment (yoloOnTheBattlefield) [inline src/utils/MarketCap/index.js]: no commented out code

**Review by yoloOnTheBattlefield — COMMENTED** [review]
https://developer-pro.mykajabi.com/products/react-pro/categories/2149030003/posts/2152230042

**Comment by keiryan** [comment]
### Pull Request 2 Change log:

- Implemented coin API
- Updated absolute imports in App.js
- Added bold coin name styling
- Fixed number abbreviator algorithm
- Coin percentages have been moved to`<abbr>` tags to reduce space taken overall. The full percentage can be seen by hovering over the desired percentage.
- Barchart numbers have also been moved to`<abbr>` tags if full number is desired to be seen.
- Scroll bar has been set to a custom color to reduce intrusion.
- Removed all `console.log()`'s
- Removed unnecessary commented out code

File: src/components/Navbar/navbar.styles.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import styled from "styled-components";
-import {Base} from '../../app.styles'
-import {Link} from 'react-router-dom'
+import { Base } from '../../app.styles'
+import { Link } from 'react-router-dom'
```
Comment (yoloOnTheBattlefield) [inline src/components/Navbar/navbar.styles.js:2]: absolute before relative

File: src/pages/Coinpage/coinpage.js [hunk 4]
```diff
@@ -1,11 +1,24 @@
 import React from "react";
+import CoinContainer from '../../components/CoinContainer/coincontainer'
+import { MiddleContainer } from './coinpage.styles'
+import axios from 'axios';
```
Comment (yoloOnTheBattlefield) [inline src/pages/Coinpage/coinpage.js:2]: absolute before relative

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

**Pull request merged**